__tests__/__packages__/cli-test-utils/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+All notable changes to the Zowe CLI test utils package will be documented in this file.
+
+## Recent Changes
```

Comment:
Did we say we are going to go with a fixed versioning scheme (likely 6.31.0 or 7.0.0-next I think)?

Reply:
I think we're kind of forced to do so the way the package is currently set up in the monorepo 😢

Reply:
thanks!